projects/extension/sql/idempotent/012-vectorizer-api.sql

```diff
@@ -106,8 +106,15 @@ begin
     -- validate the scheduling config
     perform ai._validate_scheduling(scheduling);
 
+    -- validate the processing config
     perform ai._validate_processing(processing);
 
+    -- if scheduling is none then indexing must also be none
+    if scheduling operator(pg_catalog.->>) 'implementation' = 'none'
+    and indexing operator(pg_catalog.->>) 'implementation' != 'none' then
+        raise exception 'automatic indexing is not supported without scheduling. indexing must be none if scheduling is none';
```

Comment:
Sorry, should we be more clear and display `indexing=>ai.indexing_none()`?

Reply:
updated. look okay to you @alejandrodnm ?